app/views/inventories/index.html.erb

```diff
@@ -0,0 +1,68 @@
+<div class="container-fluid">
+  <div class="row p-3 bg-table">
+    <h3>Manage Inventory</h3>
+    <div class="ml-auto">
+      <button class="btn btn-sm btn-success"> <span class="icon material-icons"> add_circle </span> Add New
+        Item</button>
+    </div>
+  </div>
+  <div class="table-responsive">
+    <table class="table table-hover">
+      <thead>
+        <tr>
+          <th scope="col">Item ID</th>
+          <th scope="col">Name</th>
+          <th scope="col">Wharehouse</th>
+          <th scope="col">State</th>
+          <th scope="col">Actions</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td>331</td>
+          <td>Ball</td>
+          <td>8d87887b</td>
+          <td>Avaliable</td>
+          <td class="text-center">
+            <span class="material-icons icon-edit">create</span>
+            <span class="material-icons icon-delete">delete</span>
+          </td>
+        </tr>
+        <tr>
+          <td>331</td>
+          <td>Refined Concrete Table</td>
+          <td>8d87887b</td>
+          <td>Avaliable</td>
+          <td class="text-center">
+            <span class="material-icons icon-edit">create</span>
+            <span class="material-icons icon-delete">delete</span>
+          </td>
+        </tr>
+        <tr>
+          <td>331</td>
+          <td>Handcrafted Soft Pizza</td>
+          <td>8d87887b</td>
+          <td>Avaliable</td>
+          <td class="text-center">
+            <span class="material-icons icon-edit">create</span>
+            <span class="material-icons icon-delete">delete</span>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+
+  <nav aria-label="Page navigation example">
+    <ul class="pagination justify-content-end mr-2">
+      <li class="page-item disabled">
+        <a class="page-link" href="#" tabindex="-1" aria-disabled="true">Previous</a>
+      </li>
+      <li class="page-item"><a class="page-link" href="#">1</a></li>
+      <li class="page-item"><a class="page-link" href="#">2</a></li>
+      <li class="page-item"><a class="page-link" href="#">3</a></li>
+      <li class="page-item">
+        <a class="page-link" href="#">Next</a>
```

Comment:
just try to avoid the `href` without a link

Reply:
Thanks, I'll replace the a tag for link_to when adding pagination functionality, Could you tell me which is the common approach? or some link to read, I always use tables  and I want to improve the code